network/OpenSSL/OpenSSLConnection.hpp

```diff
@@ -105,7 +105,7 @@ namespace awsiotsdk {
             bool certificates_read_flag_;
             bool enable_alpn_;
 
-            bool ipv6_connection;                       ///< Connect using ipv6
+            int address_family;                       ///< Should be AF_INET or AF_INET6
```

Comment:
Should be ```address_family_``` for consistency. Otherwise, looks good to go!

Reply:
Fixed